.github/workflows/stale-community-issues.yaml

```diff
@@ -0,0 +1,26 @@
+name: Stale issue action
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          any-of-labels: "community"
+          exempt-issue-labels: "frozen"
```

Comment:
I feel like maybe there are some more exempt ones here and below. The ones that comes to mind the ones that are still on the roadmap. https://github.com/orgs/airbytehq/projects/37/views/1
We could run scripts to tag those automatically.

Reply:
I'm not sure if there are multiple tags here like `roadmap` and `somethingelse` or there should just be one like `evergreen` or something.

Reply:
Roadmap issues are usually created by the Airbyte team and won't have the `community` label.

Reply:
why is `frozen` exempt though. Just so it doesn't compete with the other timeline? What about ones from the community that are ebing worked on. Maybe those get assigned and stuff so they don't get stale?